apidef/oas/default.go

```diff
@@ -38,6 +40,7 @@ type TykExtensionConfigParams struct {
 	UpstreamURL     string
 	ListenPath      string
 	CustomDomain    string
+	ApiId           string
```

Comment:
shall we name it `ApiID`, acronym with two letters

Reply:
Also there could be cases when `ApiID` isn't necessary. For eg, for usage from dashboard. shall we treat this as a ptr like what we do for `Authentication`, `AllowList`, `ValidateRequest`

Reply:
I'm not really sure about pointers, we ought to find another way